libs/expo/betterangels/src/lib/helpers/generatePublicNote.ts

```diff
@@ -21,7 +20,7 @@ export default function generatePublicNote(watchedValues: IWatchedValue) {
         ? 'The goal for this session was to'
         : 'The goals for this session were to'
       : '';
-  const changedP = nextSteps
+  const changedP = [{ title: '' }]
```

Comment:
wouldn't this always resolve to `[ ]`?
then this may impact the `updatedP` definition

Reply:
https://github.com/BetterAngelsLA/monorepo/pull/581

upcoming PR resolves that issue. that is why just left placeholder